view/css/main.css

```diff
@@ -19,7 +19,7 @@ header {
     max-height: 100vh;
     border-right: 2px solid #6d5e5e;
     position: absolute;
-    width: 230px;
+    width: 400px;
```

Comment:
please avoid hard coding. Use a percentage or **vh** property, since they are screen dependent. Using px would make screens buggy on lower resolutions of HD.